app/portainer/views/endpoints/edit/endpoint.html

```diff
@@ -210,6 +212,63 @@
             </div>
             <por-endpoint-security form-data="formValues.SecurityFormData" endpoint="endpoint"></por-endpoint-security>
           </div>
+          <!-- open-amt info (settings.FeatureFlagSettings && settings.FeatureFlagSettings['open-amt']) -->
+          <div ng-if="state.managementinfo !== ''">
+            <div class="col-sm-12 form-section-title">
+              Open Active Management Technology
+            </div>
+            <div class="form-group">
+              <label for="endpoint_managementinfo" class="col-sm-3 col-lg-2 control-label text-left">
+                Debug Info
+                <portainer-tooltip position="bottom" message="AMT Debug.."> </portainer-tooltip>
+              </label>
+              <div class="col-sm-9 col-lg-10">
+                <input
+                  type="text"
+                  ng-disabled="true"
+                  class="form-control"
+                  id="endpoint_managementinfo"
+                  ng-model="endpoint.ManagementInfoRaw"
+                  placeholder="querying node AMT info..."
+                />
+              </div>
+            </div>
+
+            <div class="form-group">
+              <label for="endpoint_managementinfoVersion" class="col-sm-3 col-lg-2 control-label text-left">
+                AMT Version
+                <portainer-tooltip position="bottom" message="AMT Version"> </portainer-tooltip>
+              </label>
+              <div class="col-sm-9 col-lg-10">
+                <input
+                  type="text"
+                  ng-disabled="true"
+                  class="form-control"
+                  id="endpoint_managementinfoVersion"
+                  ng-model="endpoint.ManagementInfo.AMT"
```

Comment:
just for consistency with the bellow row
```suggestion
                  ng-model="endpoint.ManagementInfo['AMT']"
```